library/Icingadb/Widget/ItemList/RedundancyGroupListItem.php

```diff
@@ -0,0 +1,153 @@
+            ->with($relations)
+            ->filter($filter);
+
+        $members = RedundancyGroup::on($this->getDb())
+            ->columns([
+                'id' => 'id',
+                'parent_output' => new Expression(
+                    'CASE WHEN redundancy_group_from_to_host_state.output IS NULL'
+                    . ' THEN redundancy_group_from_to_service_state.output'
+                    . ' ELSE redundancy_group_from_to_host_state.output END'
+                ),
+                'parent_long_output' => new Expression(
+                    'CASE WHEN redundancy_group_from_to_host_state.long_output IS NULL'
+                    . ' THEN redundancy_group_from_to_service_state.long_output'
+                    . ' ELSE redundancy_group_from_to_host_state.long_output END'
+                ),
+                'parent_checkcommand_name' => new Expression(
+                    'CASE WHEN redundancy_group_from_to_host.checkcommand_name IS NULL'
+                    . ' THEN redundancy_group_from_to_service.checkcommand_name'
+                    . ' ELSE redundancy_group_from_to_host.checkcommand_name END'
+                ),
+                'parent_last_state_change' => new Expression(
+                    'CASE WHEN redundancy_group_from_to_host_state.last_state_change IS NULL'
+                    . ' THEN redundancy_group_from_to_service_state.last_state_change'
+                    . ' ELSE redundancy_group_from_to_host_state.last_state_change END'
+                ),
+                'parent_severity' => new Expression(
+                    'CASE WHEN redundancy_group_from_to_host_state.severity IS NULL'
+                    . ' THEN redundancy_group_from_to_service_state.severity'
+                    . ' ELSE redundancy_group_from_to_host_state.severity END'
+                )
+            ])
+            ->with($relations)
+            ->filter($filter)
+            ->orderBy([
+                'parent_severity',
+                'parent_last_state_change',
+            ], 'DESC');
+
+        $this->applyRestrictions($members);
+
+        /** @var RedundancyGroup $data */
+        $data = $members->first();
+
+        if($data) {
+            $caption->addHtml(new PluginOutputContainer(
+                (new PluginOutput($data->parent_output . "\n" .$data->parent_long_output))
+                    ->setCommandName($data->parent_checkcommand_name)
+            ));
+        }
+
+        $caption->addHtml(new RedundancyGroupStatistics($summary->first()));
+    }
+
+    protected function assembleTitle(BaseHtmlElement $title): void
+    {
+        if ($this->state->failed) {
+            $verb = 'has';
+        } else {
+            $verb = 'is';
+        }
+
+        $title->addHtml(Html::sprintf(
+            t('%s %s %s', '<hostname> has/is <state-text>'),
+            $this->createSubject(),
+            $verb,
+            Html::tag('span', ['class' => 'state-text'], $this->state->failed ? 'Failed' : 'OK')
```

Comment:
Translatable strings should never be concatenated. Instead, the text to translate should always be complete.